core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -430,6 +442,7 @@ impl TransactionViewReceiveAndBuffer {
                     if result.is_err() {
                         num_dropped_on_status_age_checks += 1;
                         container.remove_by_id(priority_id.id);
+                        continue;
```

Comment:
this is a fix for the recent panic; change is isolated and labeled for backport in #5180